dds/DCPS/DataReaderImpl_T.h

```diff
@@ -2330,35 +2331,31 @@ class FilterDelayedHandler : public Watchdog {
 
   virtual void reschedule_deadline()
   {
-    if (!data_reader_var_.in()) {
-      return;
-    }
-
-    ACE_GUARD(ACE_Recursive_Thread_Mutex, guard, data_reader_impl_->sample_lock_);
+    if (data_reader_impl_) {
```

Comment:
Why is this changing from "check a condition and return" to putting the whole method within an "if"?  Seems like the former way was fine.  (Same for cleanup() below.)